src/content-script.ts

```diff
@@ -36,6 +36,14 @@ import { isBodyWithReplies, isRepositoryDetails } from "./validations";
 	// 4. Fetch the repository's .github/replies.yml
 	const repliesUrl = `https://raw.githubusercontent.com/${userOrOrganization}/${repository}/${defaultBranch}/.github/replies.yml`;
 	const repliesResponse = await fetch(repliesUrl);
+
+	if (!repliesResponse.ok) {
+		if (repliesResponse.status !== 404) {
+			console.error("Fetch replies:", repliesResponse.statusText);
```

Comment:
[Nitpicking] Just making this a bit more understandable to layfolk:

```suggestion
			console.error("Non-ok response fetching replies:", repliesResponse.statusText);
```